akka-actor/src/main/scala/akka/serialization/SerializationExtension.scala

```diff
@@ -4,14 +4,23 @@
 
 package akka.serialization
 
+import akka.actor.ClassicActorSystemProvider
 import akka.actor.{ ActorSystem, ExtendedActorSystem, ExtensionId, ExtensionIdProvider }
 
 /**
  * SerializationExtension is an Akka Extension to interact with the Serialization
  * that is built into Akka
  */
 object SerializationExtension extends ExtensionId[Serialization] with ExtensionIdProvider {
+  override def apply(system: ActorSystem): Serialization = super.apply(system)
```

Comment:
this is not needed, but I also tried with apply(ClassicActorSystemProvider) and that doesn't work: ambiguous reference to overloaded definition

Reply:
Wouldn't we just change `def apply(system: ActorSystem)` in `Extension` to take an `ClassicActorSystemProvider`?

Reply:
Would be nice, but I’d be suprised if that would be bin compat.

Reply:
I'm trying to add an overload like this in #28294 which seems to work.

Reply:
I'm trying to add an overload like this in #28294 which seems to work.